### ISSUE TITLE
Cannot find EncryptionUtils

### ISSUE THREAD

**Issue by Rudloff** [kalaspuffar/secure-quick-reliable-login#183]
### Issue or error solving.

*Short description of the issue*
Building the app (ca23d8c14a2cfc83889bb452c4e409e711b0d465) with Gradle fails.

*Expected behavior*
The app builds correctly.

*Actual behavior*
```
> Task :app:compileDebugJavaWithJavac FAILED
/home/pierre/www/secure-quick-reliable-login/app/src/main/java/org/ea/sqrl/activites/TextImportActivity.java:94: error: cannot find symbol
                        int line = EncryptionUtils.getInteger(e.getMessage());
                                                  ^
  symbol:   method getInteger(String)
  location: class EncryptionUtils
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: /home/pierre/www/secure-quick-reliable-login/app/src/main/java/org/ea/sqrl/activites/LoginBaseActivity.java uses unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
1 error
```

*Steps to reproduce*
`./gradlew assemble`

**Comment by kalaspuffar** [comment]
Hi @Rudloff 

Thanks for the report. Seems my last merge went side ways. Didn't have time to actually work today so I just merged the code to get the last changes in and sadly a compile error sneaked in. I've fixed this issue in the next commit.

Thanks for helping out and compiling the code. To be sure that the code compiles choose a version called Bumped version because that's equal to a release version. Next release I will tag as well for ease of use.

Best regards
Daniel
